### ISSUE TITLE
Add-on compatibility

### ISSUE THREAD

**Issue by drinfernoo** [cartmandos/skin.titan.bingie.mode#3]
The Netflix views in this skin are awesome! However... they don't seem to work very well at all without library content :(

The TV Show info dialog shows "More episodes", for example, but without library content, there's no episodes loaded :(

I've also had trouble getting trailers to play automatically on home... even though they do in the info dialog. What's causing this incompatibility?

**Comment by cartmandos** [comment]
It's the playmedia command that must get the trailer link/address from kodi's internal db.
to change that behavior all playmedia commands has to be replace with playtraileryoutube method I wrote in skin helper, but this in turn will take more calculation time.

on the ''more episodes'' button issue, it looks for the dbid content, which obvously doesn't exist, to bypass this you could use ''sendclick'' command with the id of kodi's default play (or 'enter' if you will), I think it's ID=9.

I'll look into it when I have the time, maybe I'll add an option to switch the trailer looking sources globally and in tvshows add the default ''play'' button as backup for non-library content.

**Comment by drinfernoo** [comment]
Those sound like reasonable fixes. I'll keep an eye out.

**Comment by SerpentDrago** [comment]
Yes it would be great if Auto Trailers could work without having to have Proper Library content

**Comment by drinfernoo** [comment]
I found a "bug" maybe, but it actually provides the desired result. Backing out of the Info dialog just as the trailer is about to play, makes it play in the Spotlight on the home screen.

To reproduce:

From the home screen, open the Info dialog on an item (movie or TV show).
Just as the screen turns black when its about to play the trailer, back out.

The trailer will then play in the Spotlight. This happens in the newest version available from bingie repository, with no modifications. It happens with library content as well as content from add-ons.

I've only found this to work with one particular add-on (a private one, still in development), but not with others, unfortunately. When trying to reproduce with other add-ons, the trailer simply filled the whole screen, rather than just the Spotlight.

**Comment by drinfernoo** [comment]
My previous comment had to do with content added as .strm files. I've now tried with local content, and I'm getting no YouTube lookup at all for shows, and no trailers playing at all, as I haven't added any local trailers.

**Comment by cartmandos** [comment]
As mentioned in the issue 'Question Regarding Spotlight', changes has been implemented in v0.1.4 that will soon release, let me know if that works for you.

ps the ''bug'' you mentioned is the intended behavior :)
